### DISCUSSION TITLE
Which Examples for the Gallery?

### DISCUSSION THREAD

**Discussion by tpike3** [mesa/mesa#1980] [Q&A]
@ankit50 is building a gallery of examples based on @rlskoeser's https://simrisk.pulcloud.io/riskybet.  We assess  5-7 examples is about right. There is two parts to this discussion 
Part 1-  Seminal Models vs Feature Split 
Part 2 - Which examples?

To start the discussion: 

Part 1 - Split 2 seminals; 5 features 
Part 2 - 
1. Schelling
2. Sugarscape g1mt
3. Caching and Replaying
4. Virus on a Network
5. Charts
6. Shape Example
7. Color Patches

Lest us know what you think

**Comment by quaquel** [comment]
What about wold sheep and flocking? Those are now part of the benchmarking as well.

Virus on a network might be a good place to point to [the ema workbench example](https://emaworkbench.readthedocs.io/en/latest/examples/example_mesa.html) for how to do analysis beyond what is covered by the batch_run functionality.
